build_tools/requirements_conda.txt

```diff
@@ -7,3 +7,4 @@ nltk==3.2.1
 pytest==2.9.1       
 flask==0.12         # server
 requests==2.11.1
+matplotlib==1.5.3
```

Comment:
Let's rather run a `pip install -r build_tools/requirements_extra_pip.txt` before running the examples in Travis CI and Appveyor (but after running unit tests)? It would be best not to add more hard dependencies (cf issue #7 ), even though this function would indeed require matplotlib..